File: 프로그래머스/Level1/2016년.py
# https://programmers.co.kr/learn/courses/30/lessons/12901

def solution(a, b):
    months = [0,31,29,31,30,31,30,31,31,30,31,30,31]
    days = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    total_days = sum(months[1 : a]) + b - 1
    answer = days[total_days % 7]
    return answer

File: 프로그래머스/Level1/2016년.js
// https://programmers.co.kr/learn/courses/30/lessons/12901

const solution = (a, b) => {
	const months = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
	let day = b;
	for (let i = 1; i < a; i++) {
		day += months[i];
	}
	const days = ['THU', 'FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED'];
	return days[day % 7];
};
